### PR TITLE
Fix main description in G8sControlPlane

### PR DESCRIPTION
**Pull request by marians**
This fixes a mistake in the main description of G8sControlPlane.

## Checklist

- [ ] Consider SIG UX feedback.
- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Change `release` CR back to be cluster scoped.
+- Fixed mistake in the main description of `G8sControlPlane`
```
Comment (xh3b4sd) [inline CHANGELOG.md:19]: There are conversations going on what users need to see in a changelog and I am not sure this is super useful information to have. Of course fixing a bug should be reflected but I would question how much bug we have here at hand really. Your call. 

```suggestion

### Fixed 

- Fix mistake in the main description of `G8sControlPlane`.
```

Comment (kopiczko) [inline CHANGELOG.md:19]:
```suggestion

### Fixed

- Fix mistake in the main description of `G8sControlPlane`.
```

Comment (marians) [reply CHANGELOG.md]: Already fixed through Tim's comment.

File: CHANGELOG.md [hunk 2]
```diff
@@ -22,6 +22,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 
 
+### Fixed 
+
+- Fix mistake in the main description of `G8sControlPlane`.
```
Comment (kopiczko) [inline CHANGELOG.md:27]:
```suggestion
- Fix mistake in the main description of `G8sControlPlane` in `infrastructure.giantswarm.io`.
```

**Pull request merged**